modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -557,6 +557,20 @@ pub struct WalletClientContext {
 impl Context for WalletClientContext {}
 
 impl WalletClientModule {
+    fn get_rpc_config(cfg: &WalletClientConfig) -> BitcoinRpcConfig {
+        if let Ok(rpc_config) = BitcoinRpcConfig::from_env_vars() {
+            // TODO: Wallet client cannot support bitcoind RPC until the bitcoin dep is
+            // updated to 0.30
+            if rpc_config.kind != "bitcoind" {
+                rpc_config
+            } else {
+                cfg.default_bitcoin_rpc.clone()
+            }
```

Comment:
Not sure we should do this, can't we just fail and tell the user that what they intended to do didn't work?

Reply:
Yes, unfortunately it breaks our developer environment since we use the same environment variable on the server side, which is set to `bitcoind` in devimint. I could use different environment variables for the client?

Reply:
never mind, we'll hopefully be upgrading bitcoin soon anyway